packages/client-sdk-web/test/integration/shared/vector-data-plane.test.ts

```diff
@@ -0,0 +1,6 @@
+import {SetupVectorIntegrationTest} from '../integration-setup';
+import {runVectorDataPlaneTest} from '@gomomento/common-integration-tests';
+
+const {vectorClient} = SetupVectorIntegrationTest();
+
+runVectorDataPlaneTest(vectorClient);
```

Comment:
I will review the rest of the bit once you rebase the PR as you mentioned on the description. Just adding a note that we should probably ignore this suite from the `integration-test` directive and add a new directive for this as well similar to how we did for `control`. You'll also need to update `runJavascriptIntegrationTests` in [canaries-kitchenshink](https://github.com/momentohq/canaries-kitchensink/blob/main/canaries/sdk-canary/index.ts#L1370) for the canaries to pick 'em up.